bindings/azure/servicebusqueues/metadata.go

```diff
@@ -0,0 +1,124 @@
+package servicebusqueues
+
+import (
+	"encoding/json"
+	"errors"
+	"strings"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+
+	"github.com/dapr/components-contrib/bindings"
+	contrib_metadata "github.com/dapr/components-contrib/metadata"
+)
+
+type serviceBusQueuesMetadata struct {
+	ConnectionString           string `json:"connectionString"`
+	NamespaceName              string `json:"namespaceName,omitempty"`
+	QueueName                  string `json:"queueName"`
+	TimeoutInSec               int    `json:"timeoutInSec"`
+	MaxConnectionRecoveryInSec int    `json:"maxConnectionRecoveryInSec"`
+	MinConnectionRecoveryInSec int    `json:"minConnectionRecoveryInSec"`
+	MaxRetriableErrorsPerSec   *int   `json:"maxRetriableErrorsPerSec"`
+	MaxActiveMessages          int    `json:"maxActiveMessages"`
+	LockRenewalInSec           int    `json:"lockRenewalInSec"`
+	MaxConcurrentHandlers      int    `json:"maxConcurrentHandlers"`
+	ttl                        time.Duration
+}
+
+const (
+	// Default time to live for queues, which is 14 days. The same way Azure Portal does.
+	defaultMessageTimeToLive = time.Hour * 24 * 14
+
+	// Default timeout in seconds
+	defaultTimeoutInSec = 60
+
+	// Default minimum and maximum recovery time while trying to reconnect
+	defaultMinConnectionRecoveryInSec = 2
+	defaultMaxConnectionRecoveryInSec = 300
+
+	// Default lock renewal interval, in seconds
+	defaultLockRenewalInSec = 20
+
+	// Default number of max active messages
+	// Max active messages should be >= max concurrent handlers
+	defaultMaxActiveMessages = 1
```

Comment:
This and defaultMaxConcurrentHandlers are much lower than in the pubsub component to maintain compatibility with the old behavior, which was synchronous and processed one message at a time only